zwave-classifier.js

```diff
@@ -809,23 +785,18 @@ class ZWaveClassifier {
   }
 
   addCarbonMonoxideLevelProperty(node, carbonMonoxideValueId) {
-    const descr = {
-      '@type': 'CarbonMonoxideLevelProperty',
-      label: 'Carbon Monoxide Level',
-      type: 'number',
-      readOnly: true,
-    };
-    descr.unit = 'ppm';
     this.addProperty(
       node,
       'carbonMonoxideLevel',
-      descr,
+      {
+        '@type': 'LevelProperty',
+        label: 'Carbon Monoxide Level',
```

Comment:
LevelProperty requires a minimum and maximum: https://iot.mozilla.org/schemas/#LevelProperty

If there isn't a logical min/max, then I would drop the @type and then it will show up as a number due to the 'type': number. You can use multipleOf: to control the number of digits after the decimal place.